packages/code-du-travail-data/dataset/fiches_ministere_travail/package.json

```diff
@@ -8,15 +8,17 @@
     "url": "https://github.com/SocialGouv/code-du-travail-numerique/issues"
   },
   "dependencies": {
+    "@cdt/data": "^3.5.0",
+    "@cdt/sources": "^3.5.0",
     "google-spreadsheets": "^2.0.0",
     "jsdom": "^15.2.1",
     "ora": "^4.0.2",
     "p-limit": "^2.2.1",
     "superagent": "^5.1.0"
   },
   "scripts": {
-    "prestart": "node verifyUrl.js",
-    "start": "node index.js > fiches-mt.json",
-    "updateTheme": "node updateTheme.js > out.json && mv out.json fiches-mt.json && node report.js"
+    "verifyUrl": "node verifyUrl",
```

Comment:
why dit you remove it from prestart ? it on pupise that this script is called before updating content so we can know about potential url changes